jba/tests/processing/test_tasktracker_content_collector.py

```diff
@@ -0,0 +1,55 @@
+import os
+import sys
+from pathlib import Path
+
+from core.src.utils.subprocess_runner import run_in_subprocess
+from jba.src import MAIN_FOLDER
+
+from jba.tests.processing import PROCESSING_FOLDER
+
+import tempfile
+
+import yaml
+
+from deepdiff import DeepDiff  # noqa: SC200
+
+
+def test_incorrect_arguments():
+    command = [sys.executable, (MAIN_FOLDER.parent / 'processing' / 'tasktracker_content_collector.py')]
+
+    stdout, stderr = run_in_subprocess(command)
+
+    assert stdout == ''
+    assert 'error: the following arguments are required' in stderr
+
+
+def yaml_as_dict(my_file):
+    result = {}
+    with open(my_file, 'r') as file_path:
+        docs = yaml.safe_load_all(file_path)
+        for doc in docs:
+            for key, value in doc.items():
+                result[key] = value
+    return result
```

Comment:
You can use `parse_yaml` from `yaml_utils.py`

Reply:
I need to use yaml.safe_load_all while parse_yaml() uses yaml.safe_load